src/time_unit/mod.rs

```diff
@@ -258,4 +248,27 @@ where
             }
         }
     }
+
+    fn ordinals_from_root_specifier(root_specifier: &RootSpecifier) -> Result<OrdinalSet> {
+        let ordinals = match root_specifier {
+            RootSpecifier::Specifier(specifier) => Self::ordinals_from_specifier(&specifier)?,
+            RootSpecifier::Period(start, step) => {
+                let base_set = match start {
+                    // A point prior to a period implies a range whose start is the specified
+                    // point and terminating inclusively with the inclusive max
+                    Specifier::Point(start) => {
+                        let start = Self::validate_ordinal(*start)?;
+                        (start..Self::inclusive_max() + 1).collect()
```

Comment:
You can use `..=` for an inclusive range, which is slightly more self-documenting.

```suggestion
                        (start..=Self::inclusive_max()).collect()
```

Reply:
Good call 👍